internal/binutils/testdata/hello.c

```diff
@@ -0,0 +1,6 @@
+#include <stdio.h>
+
+int main() {
+  printf("Hello, world!\n");
+  return 0;
+}
```

Comment:
end of line is missing

Reply:
Is there a linter that should've caught this?

Reply:
I'm not well familiar with C/C++ OSS linters and it's probably not worth the effort integrating it into this repo.